system/jlib/jlzbase.hpp

```diff
@@ -200,7 +200,7 @@ class CStreamCompressor : public CSimpleInterfaceOf<ICompressor>
     size32_t outputExtra = 0;
 
     //Options for configuring the compressor:
-    byte maxCompression = 20;   // Avoid compressing more than 20x because allocating when expanding is painful.
+    byte maxCompression = defaultMaxCompressionFactor;   // Avoid compressing more than 50x because allocating when expanding is painful.
```

Comment:
could change 50x text to something like "a significant amount" or "set limit" ...